sensibility/miner/corpus.py

```diff
@@ -121,6 +121,8 @@ def __init__(self, engine=None,
                 if path is None:
                     path = get_sqlite3_path() if path is None else path
                 url = f"sqlite:///{os.fspath(path)}"
+            if not os.path.exists(os.path.dirname(path)):
```

Comment:
I generally use [pathlib](https://docs.python.org/3/library/pathlib.html) for these kinds of manipulations, but this looks good for me!